main.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/gin-gonic/contrib/static"
 	"github.com/gin-gonic/gin"
 	"github.com/johscheuer/todo-app-web/tododb"
-	"github.com/prometheus/client_golang/prometheus"
+	"github.com/mcuadros/go-gin-prometheus"
```

Comment:
The go-gin-prometheus middleware collect metrics about the HTTP Handler and requests.
```
# HELP gin_request_duration_seconds The HTTP request latencies in seconds.
# TYPE gin_request_duration_seconds summary
gin_request_duration_seconds{quantile="0.5"} NaN
gin_request_duration_seconds{quantile="0.9"} NaN
gin_request_duration_seconds{quantile="0.99"} NaN
gin_request_duration_seconds_sum 0
gin_request_duration_seconds_count 0
# HELP gin_request_size_bytes The HTTP request sizes in bytes.
# TYPE gin_request_size_bytes summary
gin_request_size_bytes{quantile="0.5"} NaN
gin_request_size_bytes{quantile="0.9"} NaN
gin_request_size_bytes{quantile="0.99"} NaN
gin_request_size_bytes_sum 0
gin_request_size_bytes_count 0
# HELP gin_response_size_bytes The HTTP response sizes in bytes.
# TYPE gin_response_size_bytes summary
gin_response_size_bytes{quantile="0.5"} NaN
gin_response_size_bytes{quantile="0.9"} NaN
gin_response_size_bytes{quantile="0.99"} NaN
gin_response_size_bytes_sum 0
gin_response_size_bytes_count 0
```